src/lesson3/Additionaly/ADD3.java

```diff
@@ -0,0 +1,71 @@
+package lesson3.Additionaly;
+
+public class ADD3 {
+    public static void main(String[] args) {
+
+
+        for (int a1 = 1; a1 <= 10; a1++) {
```

Comment:
Very very strange!!!
1. Why do you use separate loop for each number?
2. Why do you use `break`, if ypu can change the loop condition a bit?
3. Formating